Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -457,7 +463,7 @@ fragment float4 fragmentShader(RasterizerData in [[stage_in]],
 
     void Camera::Impl::Close()
     {
-        m_implData.reset();
+        m_implData.reset(new ImplData);
```

Comment:
Why does this have to be destroyed? And if it does have to be destroyed, can we instead do `m_implData = std::make_unique<ImplData>()`?